src/main/java/nl/dtls/fairdatapoint/api/config/RestApiContext.java

```diff
@@ -198,8 +199,23 @@ public Repository repository(@Value("${store.type:1}") int storeType)
     }
     
     @Bean
-    public PIDSystem pIDSystem() {
-        return new DefaultPIDSystemImpl();
+    @DependsOn({"purlBaseUrl"})

```

Comment:
Is the `@DependsOn` really needed here? I assume Spring will figure out the dependency graph on its own, making this redundant. Or is it needed if there are multiple competing beandefinitions available in the context?